packages/header-parser/src/index.ts

```diff
@@ -1,42 +1,32 @@
-import pm = require("parsimmon");
 import { AllTypeScriptVersion, TypeScriptVersion } from "@definitelytyped/typescript-versions";
+import assert = require("assert");
+import * as semver from "semver";
+import { deepEquals } from "@definitelytyped/utils";
 
-/*
-
-  # Example header format #
-
-  // Type definitions for foo 1.2
-  // Project: https://github.com/foo/foo, https://foo.com
-  // Definitions by: My Self <https://github.com/me>, Some Other Guy <https://github.com/otherguy>
-  // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-  // TypeScript Version: 2.1
-
-*/
-
+// used in dts-critic
 export interface Header {
   readonly nonNpm: boolean;
-  readonly libraryName: string;
+  readonly nonNpmDescription?: string;
+  readonly name: string;
   readonly libraryMajorVersion: number;
   readonly libraryMinorVersion: number;
   readonly typeScriptVersion: AllTypeScriptVersion;
   readonly projects: readonly string[];
   readonly contributors: readonly Author[];
 }
+// used in definitions-parser
+export type Author =
```

Comment:
Annoyingly we're going to be calling this "owner" now. Oh well.

Reply:
Yeah however you want, I am not picky. Don't know who all is using this code but I'm sure this change is breaking in every conceivable way to rename how you feel.